src/styles/index.scss

```diff
@@ -1,3 +1,221 @@
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+
+  &__logo {
+    display: flex;
+
+    &__img {
+      display: block;
+    }
+  }
+}
+
+.nav {
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-end;
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    text-transform: uppercase;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    position: relative;
+    text-decoration: none;
+    color: black;
+    font-size: 12px;
+    padding: 0;
+    text-align: center;
+    height: 60px;
+    line-height: 60px;
+    margin-left: 20px;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+}
+
+.is-active {
+  color: #00acdc;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 4px;
+    background-color: #00acdc;
+    border-radius: 8px;
+  }
+}
+
+.card {
+  width: var(--card-width);
+  padding: 16px;
   margin: 0;
+  display: grid;
+
+  &__qwe {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    width: 100%;
+  }
+
+  &__image {
+    width: 160px;
+    display: flex;
+    margin-top: 12px;
+    margin-bottom: 40px;
+    justify-content: center;
+  }
+
+  &__name {
+    display: flex;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+    width: 100%;
+  }
+
+  &__product {
+    &-code {
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
+      color: #616070;
+      display: flex;
+      margin-bottom: 16px;
+    }
+  }
+
+  &__rate {
+    &-stars {
+      display: flex;
+      justify-content: center;
+      margin-bottom: 8px;
+    }
+
+    &-star {
+      width: 16px;
+      height: 16px;
+      background-image: url('../images/star.svg');
+      background-repeat: no-repeat;
+      background-position: center;
+      margin-right: 4px;
+
+      &:nth-child(-n + 5) {
+        background-image: url('../images/star-active.svg');
+      }
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    &-review {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: #060b35;
+      text-align: center;
+      margin: 0;
+    }
+  }
+
+  &__price {
+    justify-content: space-between;
+    display: flex;
+    width: 100%;
+    margin-bottom: 16px;
+
+    &-item {
+      color: #616070;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      margin: 0;
+    }
+
+    &-value {
+      color: #060b35;
+      font-weight: 700;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    background-color: #00acdc;
+    color: #f3f3f3;
+    border: none;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    text-decoration: none;
+
```

Comment:
The `.card__button:hover` should include a transition for hover effects to ensure changes are smooth over 300ms, as specified in the task requirements【12:3†source】.